ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/snmp-settings.j2

```diff
@@ -24,6 +24,24 @@
 | {{ row_contact }} | {{ row_location }} | {{ row_traps_disabled }} | Disabled |
 {%         endif %}
 {%     endif %}
+{%     if snmp_server.engine_ids is arista.avd.defined %}
+
+### SNMP EngineID Configuration
+
+| Type | EngineID (Hex) | IP | Port |
+| ---- | ---------- | -- | ---- |
+{%         if snmp_server.engine_ids.local is arista.avd.defined %}
+| local | {{ snmp_server.engine_ids.local }} | - | - |
+{%         endif %}
+{%         for engine_id in snmp_server.engine_ids.remotes %}
```

Comment:
```suggestion
{%         for engine_id in snmp_server.engine_ids.remotes | arista.avd.default([]) %}
```

Reply:
Currently you added defined test only for ```snmp_server.engine_ids``` but not ```snmp_server.engine_ids.remotes```. So when ever someone defines ```snmp_server.engine_ids``` without remotes the current code will throw error. So I suggested the above.